bears/vcs/git/GitCommitBear.py

```diff
@@ -109,6 +110,36 @@ def check_shortlog(self,
                 yield Result(
                     self,
                     "Shortlog of HEAD commit does not match given regex.")
+        colon_pos = shortlog.find(':')
+        shortlog = shortlog[colon_pos:] if colon_pos != -1 else shortlog
+        has_flaws = self.check_imperative(shortlog)
+        if has_flaws:
+            bad_words = ', '.join([words[0] for words in has_flaws])
+            yield Result(self,
+                         "Shortlog of HEAD commit isn't imperative mood,"
+                         "bad words are '{0}'".format(bad_words))
+
+    def check_imperative(self, paragraph):
+        """
+        Check the given sentence/s for Imperatives
+
+        :param paragraph: The input paragraph to be tested
+        :returns: 'Pass' or 'Fail'
+        """
+        f_sentence = nltk.sent_tokenize(paragraph)[0]
+        words = nltk.word_tokenize(f_sentence)
+        flaws = []
+        # VBZ : Verb, 3rd person singular present, like 'adds', 'writes' etc
+        # VBD : Verb, Past tense , like 'added', 'wrote' etc
+        # VBG : Verb, Present participle, like 'adding', 'writing'
+        for word, tag in nltk.pos_tag(['I'] + words)[1:]:
+            if word.endswith('ing'):    # Handle special case for VBG
+                flaws.append((word, tag))
+            elif(tag.startswith('VBZ') or
```

Comment:
Can't you just put "or" in the earlier if and remove this elif ? (They both do the same thing if true)